docs/studios/unity/unity-sdk/in-editor-replay.mdx

```diff
@@ -65,4 +65,22 @@ Shows any command(s) that the Bot sent to the game for the current tick.  These
 Shows any validation(s) that the Bot performed on the current tick.  These are normally test cases coded into the Bot to validate that the tick data changed as expected based on prior actions or other factors in the game state.  See [Validation Bots](./creating-bots/validation-bots) for more information.
 
 ### Registering Custom Models For Replay Objects
-... Coming Soon ...
+When viewing a replay, entities that are dynamically spawned into the scene (not defined in the scene itself) need to be told what model to render with.  By default they will load as a pink capsule, but they can be configured with a specific model based on their type.
```

Comment:
```suggestion
When viewing a replay, entities that are dynamically spawned into the scene (not defined in the scene itself) need to be told what model to render with. By default they will load as a pink capsule, but they can be configured with a specific model based on their type.
```

Reply:
This entire doc uses 2 spaces.  As it should be...  But I know that's a generational thing (get off my lawn :P)

I suppose I can live with 1 space after a period, but I'll need to update everything in this doc to match that style.